virt/virt-2-4-release-notes.adoc

```diff
@@ -122,6 +122,28 @@ single sidebar menu item labeled *Virtualization*. When you click *Virtualizatio
 
 [id="virt-2-4-known-issues"]
 == Known issues
+* When upgrading an {product-title} cluster from version 4.4 to 4.5, and container-native virtualization 2.3 is installed, a migrating VMI fails when the target node is evicted due to the upgrade.
```

Comment:
Something about this seems ambiguous to me, so I'm suggesting an alternative to take or leave:

`If container-native virtualization 2.3 is installed on your {product-title} 4.4 cluster, upgrading the cluster to version 4.5 causes a migrating VMI to fail when the target node is evicted during the upgrade.`

Reply:
That is much better - I'll take it!
Updated as suggested